photon-api/src/main/scala/com/linkedin/photon/ml/optimization/game/RandomEffectOptimizationProblem.scala

```diff
@@ -43,27 +43,57 @@ protected[ml] class RandomEffectOptimizationProblem[Objective <: SingleNodeObjec
     val isTrackingState: Boolean)
   extends RDDLike {
 
+  /**
+   * Get the Spark context.
+   *
+   * @return The Spark context
+   */
   def sparkContext: SparkContext = optimizationProblems.sparkContext
```

Comment:
We should replace alle `sparkContext` with `sc` across the board once and for all.